docs/backends.rst

```diff
@@ -2,7 +2,9 @@
  Proctoring backend implementation
 ===================================
 
-Proctoring services (PS) who wish to integrate with Open edX should implement a `REST API`_ and a thin `Python wrapper`_, as described below.
+Proctoring services (PS) who wish to integrate with Open edX should implement a `REST API`_ and a thin
+`Python wrapper`_, as described below.
+
```

Comment:
What prompted inserting all the newlines?  [OEP-19](https://open-edx-proposals.readthedocs.io/en/latest/oep-0019-bp-developer-documentation.html#best-practices) recommends letting paragraphs auto-wrap instead of enforcing a line-length limit in `rst` files.

Reply:
`tox -e docs` was failing, but travis doesn't test this. Hence reverted